src/files.ts

```diff
@@ -112,16 +113,17 @@ export async function getProjectFiles(rootDir: string = path.join('.', '/'), cal
 
         // check ignore files
         const ignoreMatches = multimatch(filePaths, ignorePatterns, { dot: true });
+        const intersection: string[] = filePaths.filter(file => !ignoreMatches.includes(file));
 
-        // Loop through every file.
-        const files = filePaths
+        // Loop through files that are not ignored
+        const files = intersection
           .map((name, i) => {
             const normalizedName = path.normalize(name);
 
             let type = getAPIFileType(name);
 
             // File source
-            let source = contents[i];
+            let source = fs.readFileSync(name).toString();
```

Comment:
Since we are not looping through every file, they are no longer in order. Just read the file based on name.